src/main/java/ca/yorku/cse2311/tab2pdf/ui/MainJFrame.java

```diff
@@ -195,18 +206,62 @@ public void update(String status) {
         this.STATUS_BAR.setHint(status);
 
         if (null != file) {
+
             this.STATUS_BAR.setInputFilePath(file.getAbsolutePath());   // The file we're currently editing
             this.STATUS_BAR.setSymbolsNumber(getEditorTab().getText().length()); // Size of the file
 
+
+            setTitleField();
+            setSubtitleField();
+
             // If a file is loaded, we should enable certain stuff
             this.TOOLBAR.getSaveButton().setEnabled(true);
             this.MENU_BAR.getSaveMenuItem().setEnabled(true);
             this.TOOLBAR.getExportButton().setEnabled(true);
             this.MENU_BAR.getExportMenuItem().setEnabled(true);
             getEditorTab().setEnabled(true);
+            this.EDITOR_TAB.getTitleField().setEnabled(true);
+            this.EDITOR_TAB.getSubtitleField().setEnabled(true);
+            this.EDITOR_TAB.getSpacingSlider().setEnabled(true);
+            this.EDITOR_TAB.getScalingSlider().setEnabled(true);
         } else {
             blockComponents();
         }
+
+
+    }
+    public void setTitleField(){
+        int line = 0;
+        String[] EditorContents = getEditorContents();
+        Pattern i= Pattern.compile("(\\W|^)TITLE[=](\\W|$)");
+        Matcher matcher = i.matcher("TITLE=");
+        while (matcher.find()) {
+            String string = EditorContents[line];
+            string = string.substring(6, string.length());
+            this.EDITOR_TAB.TITLE.setText(string);
+            line++;
+        }
+
+
+    }
+
+    public void  setSubtitleField(){
```

Comment:
see my notes for title parser. This method just sets the second line of the tab file as the subtitle (and fails if the second line of the text file is not `SUBTITLE=whatever`